cfgov/unprocessed/apps/agreements/Agreements.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import Select from 'react-select'
+
+function onChange(val){
+  window.location = `/credit-cards/agreements/issuer/${val.value}/`
+}
+
+function getOption(options, val){
+  for(let i=0; i<options.length; i++){
```

Comment:
Run `yarn lint`

Reply:
Local linting setup is apparently busted :^}